src/js/typescript.js

```diff
@@ -49,9 +49,15 @@ export default class TypeScriptCompiler extends SimpleCompilerBase {
 
     d(output.diagnostics);
 
+    let sourceMaps;
+    if (output.sourceMapText) {
+      sourceMaps = JSON.parse(output.sourceMapText);
```

Comment:
Does it make more sense to return this as a string instead in both places? What is the caller gonna do with this information other than write it to a file?